pkg/neotest/basic.go

```diff
@@ -182,6 +182,19 @@ func (e *Executor) InvokeScript(t *testing.T, script []byte, signers []Signer) u
 	return tx.Hash()
 }
 
+// PrepareInvokeScript creates transaction with the specified script and signs it by the provided
+// signer.
+func (e *Executor) PrepareInvokeScript(t *testing.T, script []byte, signers []Signer, validUntilBlock ...uint32) *transaction.Transaction {
```

Comment:
So why `InvokeScript` (and potentially others) doesn't reuse it?

Reply:
Fixed.